commit-data/README.md

```diff
@@ -17,6 +17,12 @@ Secondly, I often want to get the short commit SHA (conventionally the first 7 c
 
 This action takes no input, it just needs the git metadata which it gets from disk.
 
+## Inputs
+
+| Name | Description | Default |
+| ---- | ---- | ---- |
+| working_directory | Working directory to use for relative path to fetch commit data. | Defaults to current context |
```

Comment:
Working directory in which to run `git` commands. Defaults to current working directory.